lib/manageiq/appliance_console/database_configuration.rb

```diff
@@ -257,7 +257,12 @@ def self.load_current
         require 'fileutils'
         FileUtils.cp(DB_YML_TMPL, DB_YML) if File.exist?(DB_YML_TMPL)
       end
-      YAML.load_file(DB_YML)
+
+      if YAML.respond_to?(:safe_load_file)
+        YAML.safe_load_file(DB_YML, :aliases => true)
```

Comment:
@jrafanie I noticed in https://github.com/ManageIQ/manageiq-appliance_console/pull/227 you replaced `YAML.load_file` with a `File.read` + `YAML.safe_load`, it looks like `YAML.safe_load_file` accepts `:aliases => true` as well was there a reason you didn't go this route?

Reply:
:scream: okay I'll match what you have in messaging_configuration

Reply:
lol, I kept digging, kwarg for `aliases` was added in 3.1.0: https://github.com/ruby/psych/commit/4d4439d6d0adfcbd211ea295779315f1baa7dadd.  I made the assumption we'd be at 3.1.0 at least so we'll have `safe_load` with kwargs but perhaps not yet have `safe_load_file` or `unsafe_load`/`unsafe_load_file`.

Some gems support super old rubies so it was easier to just follow that pattern.  Technically, it wouldn't work with psych 3 or earlier but 🤷